### PR TITLE
Collecting review-event payload

### PR DESCRIPTION
**Pull request by hibariya**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,545 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (obatan) [inline yarn.lock]: I'm going to start a review :robot:

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,545 @@
...
+# yarn lockfile v1
+amdefine@>=0.0.4:
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/amdefine/-/amdefine-1.0.0.tgz#fd17474700cb5cc9c2b709f0be9d23ce3c198c33"
+
+async@~0.2.6:
```
Comment (hibariya) [inline yarn.lock]: single comment (another conversation)

Comment (hibariya) [inline yarn.lock]: start a review

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,545 @@
...
+    type-is "~1.5.1"
+
+brace-expansion@^1.0.0:
+  version "1.1.6"
+  resolved "https://registry.yarnpkg.com/brace-expansion/-/brace-expansion-1.1.6.tgz#7197d7eaa9b87e648390ea61fc66c84427420df9"
+  dependencies:
+    balanced-match "^0.4.1"
+    concat-map "0.0.1"
+
+broccoli-cli@0.0.1:
```
Comment (hibariya) [inline yarn.lock]: alpha

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,545 @@
...
+  version "0.0.1"
+  resolved "https://registry.yarnpkg.com/broccoli-cli/-/broccoli-cli-0.0.1.tgz#578ccbbbac36081741c03d3d38e1295bd43bcebf"
+  dependencies:
+    resolve "~0.6.1"
+
+broccoli-concat@0.0.13:
```
Comment (hibariya) [inline yarn.lock]: bravo

File: yarn.lock [hunk 5]
```diff
@@ -0,0 +1,545 @@
...
+    bytes "1.0.0"
+    depd "0.4.5"
+    iconv-lite "0.4.4"
+    media-typer "0.3.0"
+    on-finished "2.1.0"
+    qs "2.2.4"
+    raw-body "1.3.0"
+    type-is "~1.5.1"
+
+brace-expansion@^1.0.0:
```
Comment (hibariya) [inline yarn.lock]: charlie

**Comment by hibariya** [comment]
delta

File: yarn.lock (hunk 1 above)
Comment (hibariya) [inline yarn.lock]: hi

Comment (hibariya) [inline yarn.lock]: hello

**Comment by hibariya** [comment]
hi

File: yarn.lock [hunk 6]
```diff
@@ -0,0 +1,545 @@
...
+  resolved "https://registry.yarnpkg.com/broccoli-rename-files/-/broccoli-rename-files-0.0.2.tgz#e60946e2ef0c7cf9ea078cb5c8dab2f76a5a2888"
+  dependencies:
+    broccoli-filter "^0.1.6"
+    copy-dereference "^1.0.0"
+    object-assign "^1.0.0"
+
+broccoli-render-template@0.0.3:
+  version "0.0.3"
+  resolved "https://registry.yarnpkg.com/broccoli-render-template/-/broccoli-render-template-0.0.3.tgz#f8da3ddfc99929674e7562f0b7c86b7b9564ae67"
+  dependencies:
```
Comment (hibariya) [inline yarn.lock]: alice

Comment (hibariya) [inline yarn.lock]: bob

File: yarn.lock [hunk 7]
```diff
@@ -0,0 +1,545 @@
...
+
+broccoli-slow-trees@^1.0.0:
+  version "1.1.0"
+  resolved "https://registry.yarnpkg.com/broccoli-slow-trees/-/broccoli-slow-trees-1.1.0.tgz#426c5724e008107e4573f73e8a9ca702916b78f7"
+
+broccoli-static-compiler@^0.2.1:
+  version "0.2.2"
+  resolved "https://registry.yarnpkg.com/broccoli-static-compiler/-/broccoli-static-compiler-0.2.2.tgz#72c48423dbac9e41ca54808ab1f95c5af5d4a39b"
+  dependencies:
+    broccoli-kitchen-sink-helpers "^0.2.5"
```
Comment (hibariya) [inline yarn.lock]: charlie

File: yarn.lock [hunk 8]
```diff
@@ -0,0 +1,545 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+amdefine@>=0.0.4:
```
Comment (ursm) [inline yarn.lock]: hi

**Comment by ursm** [comment]
issue comment

File: yarn.lock (hunk 1 above)
Comment (ursm) [inline yarn.lock]: oh

**Comment by ursm** [comment]
issue comment again

File: yarn.lock (hunk 8 above)
Comment (ursm) [inline yarn.lock]: reply

File: yarn.lock [hunk 9]
```diff
@@ -0,0 +1,545 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (ursm) [inline yarn.lock]: yarn

File: yarn.lock [hunk 10]
```diff
@@ -0,0 +1,545 @@
...
+amdefine@>=0.0.4:
+  version "1.0.0"
```
Comment (ursm) [inline yarn.lock]: version

**Comment by ursm** [comment]
issue comment

File: yarn.lock [hunk 11]
```diff
@@ -0,0 +1,545 @@
...
+    walk-sync "^0.1.3"
+
+broccoli-kitchen-sink-helpers@^0.2.0, broccoli-kitchen-sink-helpers@^0.2.4, broccoli-kitchen-sink-helpers@^0.2.5, broccoli-kitchen-sink-helpers@^0.2.6:
+  version "0.2.9"
+  resolved "https://registry.yarnpkg.com/broccoli-kitchen-sink-helpers/-/broccoli-kitchen-sink-helpers-0.2.9.tgz#a5e0986ed8d76fb5984b68c3f0450d3a96e36ecc"
+  dependencies:
+    glob "^5.0.10"
+    mkdirp "^0.5.1"
+
+broccoli-merge-trees@^0.2.1:
```
Comment (hibariya) [inline yarn.lock]: hi

**Pull request closed without merging**